lib/scala/logging-config/src/main/java/org/enso/logging/config/systemlogger/ContextLoggingViaSlf4j.java

```diff
@@ -0,0 +1,38 @@
+package org.enso.logging.config.systemlogger;
+
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import org.enso.common.ContextLoggingConfigurator;
+import org.enso.common.LanguageInfo;
+import org.enso.logger.Converter;
+import org.enso.logger.JulHandler;
+import org.enso.logging.config.LoggerSetup;
+import org.graalvm.polyglot.Context;
+
+/**
+ * Provides implementation of the {@link ContextLoggingConfigurator} for the `engine-common` module.
+ */
+public final class ContextLoggingViaSlf4j extends ContextLoggingConfigurator {
+  public ContextLoggingViaSlf4j() {}
+
+  /** Configures provided builder via {@link LoggerSetup}. */
+  @Override
+  protected final void prepareBuilderForLogging(
```

Comment:
This method provided the same implementation that was in [ContextFactory itself](https://github.com/enso-org/enso/pull/13363/files#r2167431140). By leveling it up here we can remove dependencies of `org.enso.common` on `LoggerSetup`, etc. All of that is done here.